src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java

```diff
@@ -457,4 +463,20 @@ public String getTenantUrl() {
     public void setTenantUrl(String tenantUrl) {
         this.tenantUrl = tenantUrl;
     }
+
+    private QueryExecutor createQueryExecutor() {
+        return new QueryExecutor(this);
+    }
+
+    QueryConfigResponse getQueryConfigResponse() throws SQLException {
+        try {
+            QueryExecutor executor = createQueryExecutor();
```

Comment:
We should not make call to hyper if enableStreamFlow is false in connectionProperties

Reply:
I am passing `enable-stream-flow` in header and will make changes in QS for this.
https://github.com/rajeshkrs/Salesforce-CDP-jdbc/pull/4/files#diff-df23f4d023487b3e58cd1dce72efdd280ba081882391219cedcea789a512ed69R113